packages/react-app/src/services/savedUserQuery.service.ts

```diff
@@ -6,6 +6,10 @@ export const addQuery = async (data: SaveQueryType): Promise<SavedUserQueryProps
 	return await SavedUserQuery.create(data);
 };
 
+export const deleteQuery = async (id: string): Promise<{ ok?: number, n?: number, deletedCount?: number }> => {
+	return await SavedUserQuery.deleteOne({ _id: id });
+};
+
 export const getQueriesByDiscordId = async (query: NextApiQuery): Promise<SavedUserQueryProps[]> => {
 	const discordId = query.discordId as string;
```

Comment:
might want to check for string[]

Reply:
As we do not have multiple deletions, we can change the function name to `deleteSingleQuery`. I believe that will be more appropriate to the current user flow. WDYT?

Reply:
will do that